force-app/main/default/objects/Product_Registry__mdt/fields/Enabled__c.field-meta.xml

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<CustomField xmlns="http://soap.sforce.com/2006/04/metadata">
+    <fullName>Enabled__c</fullName>
+    <defaultValue>false</defaultValue>
+    <description>Defines if the custom metadata record is enabled</description>
+    <externalId>false</externalId>
+    <fieldManageability>DeveloperControlled</fieldManageability>
```

Comment:
I think this one should be `SubscriberControlled` so admins could disable the class

Reply:
This is one of the cases where SubscriberControlled is relevant. Leaving this open will give CCE a way to override behavior in the case of an emergency and effectively temporarily exclude a malfunctioning package, which is the purpose of this field.

Reply:
makes total sense, thanks